spec/services/bulk_upload/lettings/year2023/row_parser_spec.rb

```diff
@@ -340,8 +340,8 @@
 
           it "fetches the question's check_answer_label if it exists, otherwise it gets the question's header" do
             parser.valid?
-            expect(parser.errors[:field_19]).to eql(["You must answer q12 - address"])
-            expect(parser.errors[:field_21]).to eql(["You must answer town or city"])
+            expect(parser.errors[:field_19]).to eql(["You must answer q12 - address lines 1 and 2"])
```

Comment:
we shouldn't be lower casing these but it's not related to this ticket, more of a bulk upload bug, I'm checking if there's a ticket to fix this (if not I'll make one)